src/zenml/zen_stores/schemas/pipeline_run_schemas.py

```diff
@@ -228,12 +228,10 @@ def to_model(
                 user=self.user.to_model(True) if self.user else None,
                 schedule_id=self.schedule_id,
                 orchestrator_run_id=self.orchestrator_run_id,
-                enable_cache=self.enable_cache,
-                enable_artifact_metadata=self.enable_artifact_metadata,
```

Comment:
What about pipeline runs that already exist in the database before your change ? Shouldn't you copy these values into the pipeline configuration if not set there ?

Reply:
The pipeline configuration already had these values in it, we redundantly stored them twice in the DB before (once as field, once as part of the config)